### DISCUSSION TITLE
Is there a way to add songs to smart playlitsts after removing .nsp files?

### DISCUSSION THREAD

**Discussion by Teloshav** [navidrome/navidrome#2376] [Q&A]
I have asked this question in another place but realised that it wasn't the proper place to ask, hence why I am asking here. Hopefully I will be able to get some more information to solve my issue. :)

My personal instance of navidrome running from a docker container on my home network. I have some smart playlists that I made.

I moved the `.nsp` files outside my Navidrome folder. I moved the `.nsp` files outside the folder as I wanted to be able to add songs to those playlists, but they're not recognised whatsoever. (I try to use the `Add to playlist` button but no playlists show up. Was doing this on an administrator account.)

My idea was to get my music to be sorted into playlists from my individual folders using the smart playlists feature, which worked great, but since I got rid of them, I thought I'd be able to start modifying them.

Could anyone please explain why this is happening? I assume this is because I'm trying approach this problem in a weird way. I thought I would be free to modify the smart playlist once I got rid of the `.nsp` file.

I did manage to 'get past' this issue by exporting one of the playlists and importing that on its own, but I hope to get rid of the folder structre so then I can remove duplicate songs (which is another insidious problem I'm trying to fix, I've got 3-4 copies of some songs!! scream).

Thanks!

**Comment by deluan** [comment]
When a `.nsp` file is imported, Navidrome creates a Smart Playlist _in the DB_ for it, that is read-only by definition. Even if you remove the `.nsp` file, it is not removed/changed in the DB, so it is still a Smart Playlist. 

Currently there's no simple way to "convert" a Smart Playlist into a plain Playlist.
